source/common/ssl/context_impl.cc

```diff
@@ -527,6 +528,13 @@ ClientContextImpl::ClientContextImpl(Stats::Scope& scope, const ClientContextCon
     }
   }
 
+  if (!config.sigalgs().empty()) {
+    for (auto& ctx : tls_contexts_) {
+      int rc = SSL_CTX_set1_sigalgs_list(ctx.ssl_ctx_.get(), config.sigalgs().c_str());
+      RELEASE_ASSERT(rc == 1, "");
```

Comment:
(Note this will fail on syntax error. Is that okay for your purposes here?)

Reply:
Yeah, this is just for tests; I've done the rename suggested by @PiotrSikora, so we are good here.